app/org/sagebionetworks/bridge/services/AuthenticationServiceImpl.java

```diff
@@ -163,13 +163,12 @@ public void signUp(Study study, SignUp signUp, boolean isAnonSignUp) {
             }
             accountDao.signUp(study, signUp, isAnonSignUp);
         } catch(EntityAlreadyExistsException e) {
-            // Do not indicate to the end user that this email address has already been taken, as this 
-            // leaks information about who is in the study. Instead send a reset password request to the 
-            // email address in case user has forgotten password and is trying to sign up again.
+            // Suppress this. Otherwise it reveals if the account does not exist
+            // Non-anonymous sign ups (sign ups done by admins on behalf of users) still get a 404
             if (isAnonSignUp) {
```

Comment:
The old comments have a lot of useful information that the new comments don't. We should restore the old comments.